tests/baselines/reference/divergentAccessors1.errors.txt

```diff
@@ -0,0 +1,39 @@
+tests/cases/compiler/divergentAccessors1.ts(6,13): error TS1049: A 'set' accessor must have exactly one parameter.
+tests/cases/compiler/divergentAccessors1.ts(10,5): error TS2322: Type 'string' is not assignable to type 'number'.
+tests/cases/compiler/divergentAccessors1.ts(17,13): error TS1049: A 'set' accessor must have exactly one parameter.
+tests/cases/compiler/divergentAccessors1.ts(21,5): error TS2322: Type 'string' is not assignable to type 'number'.
+
+
+==== tests/cases/compiler/divergentAccessors1.ts (4 errors) ====
+    // Accessors in interfaces/types
+    
+    {
+        interface IHasGetSet {
+            get foo(): number;
+            set foo(): number | string;

```

Comment:
is this test meant to cover the must-have-exactly-one-parmeter error, or ...
```suggestion
            set foo(val: string);
```

with get: number | string